schema/schemata/prosecution_case.json

```diff
@@ -5,16 +5,57 @@
   "id": "schemata/prosecution_case",
   "stability": "prototype",
   "strictProperties": true,
-  "type": [
-    "object"
-  ],
+  "type": "object",
   "definitions": {
+    "resource": {
+      "description": "object representing a single prosecution case",
+      "type": "object",
+      "properties": {
+        "type": {
+          "$ref": "/schemata/prosecution_case#/definitions/type"
+        },
+        "id": {
+          "$ref": "/schemata/prosecution_case#/definitions/id"
+        },
+        "attributes": {
+          "$ref": "/schemata/prosecution_case#/definitions/attributes"
+        }
+      }
+    },
+    "attributes": {
+      "type": "object",
+      "properties": {
+        "first_name": {
+          "$ref": "/schemata/prosecution_case#/definitions/first_name"
+        },
+        "last_name": {
+          "$ref": "/schemata/prosecution_case#/definitions/last_name"
+        },
+        "prosecution_case_reference": {
+          "$ref": "/schemata/prosecution_case#/definitions/prosecution_case_reference"
+        },
+        "date_of_birth": {
+          "$ref": "/schemata/prosecution_case#/definitions/date_of_birth"
+        },
+        "national_insurance_number": {
+          "$ref": "/schemata/prosecution_case#/definitions/nino"
+        }
+      }
+    },
     "id": {
       "description": "Unique identifier of prosecution case provided by HMCTS (prosecutionCaseId)",
       "readOnly": true,
       "format": "uuid",
       "type": "string"
     },
+    "type": {
```

Comment:
are there going to be any other types than `prosecution_cases`?

Reply:
We would probably have `hearings`, `defendants`, and others defined at their other endpoints, but `type` is a reserved jsonapi keyword to identify the resource type: https://jsonapi.org/format/#document-resource-objects